.github/workflows/tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ["main"]
+  pull_request_target:
+    branches: ["main"]
+
+jobs:
```

Comment:
Maybe, lets add a workflow call to .github/workflows/ci.yaml to ensure that tests are passing before releasing an image, wdyt?